server/src/main/java/com/linecorp/centraldogma/server/internal/api/AdministrativeService.java

```diff
@@ -124,6 +124,28 @@ public ServerStatus status() {
                 }
                 return status();
             });
+        } else if (oldStatus.replicating) {
```

Comment:
How about adding a comment that describes the status change?
```
//    writable       replicating      result
//----------------------------------------------------------------------- 
//    true -> true                    not_modified exception
//-----------------------------------------------------------------------
//    true -> false  true -> true     setWritable(false)
//                   true -> false    setWritable(false) & Stop executor.
//-----------------------------------------------------------------------
//    false -> true  true -> true     setWritable(true)
//                   false -> true    setWritable(true) & start executor.
//-----------------------------------------------------------------------
//    false -> false true -> false    Stop executor.
//                   false -> true    Start executor.
```
And I think if we implement the logic in that way we can use the early return style and also we are not going to miss any cases.
Currently, I think we missed the fourth case which is `false -> true |  true -> true  |   setWritable(true)`.
Also, it would be nice if we add:
```
if (oldStatus.writable == writable && oldStatus.replicating == replicating) {
    throw HttpStatusException.of(HttpStatus.NOT_MODIFIED);
}
```